packages/react-core/src/components/Content/examples/Content.md

```diff
@@ -62,3 +62,11 @@ HTML elements wrapped by `<Content>` are styled by the content component.
 ```ts file="./ContentVisited.tsx"
 
 ```
+
+### Editorial content
+
+Editorial styling increases the font size of body text and small text by 1 tier, where body text becomes "lg" and small text becomes "md". To applying editorial styling, use `isEditorial`.
```

Comment:
This specific description of the styling always makes me a little nervous 😅 If that changes in any way, it's something we'll need to update, and in my experience this description text often gets overlooked and out of date.